comment_filter/rfc.py

```diff
@@ -160,15 +160,16 @@ def parse_code(lang, state):
     while True:
         line = state.line
         multi_start_tokens = [start for start, end in lang.comment_bookends]
-        tokens = multi_start_tokens + [
-            lang.line_comment,
+        tokens = multi_start_tokens + lang.line_comment + [
             lang.string_literal_start,
             lang.string_literal2_start]
         i = index_of_first_found(line, tokens)
         if i != -1:
             state.line = line[i:]
             code += line[:i]
-            if line.startswith(lang.line_comment, i) or \
+            # Check if any line comment delimiter starts at position i
+            line_comment_found = any(line.startswith(lc, i) for lc in lang.line_comment)
```

Comment:
```suggestion
            line_comment_found = line.startswith(tuple(lang.line_comment), i)
```
This is functionally equivalent and maybe more performant due to the iteration happening in the C implementation of startswith.

Although if we were *really* concerned about performance, we would also move the tuple conversion outside of the while loop. But I can see at least one other expression that we'd want to do that for as well, and so maybe we could save that bit for a future performance optimization PR.